Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.11-slim
+FROM python:3.13-slim
 
 WORKDIR /app
 
 RUN pip install twyn
 
-ENTRYPOINT ["twyn", "run", "-vv"]
+ENTRYPOINT ["twyn"]
```

Comment:
does this make it a breaking change?

Reply:
From the POV of the docker image: absolutely. 

Is the docker image used anywhere or by anyone? Do you think we should make it breaking because of it?

Reply:
As far as I know it does not have an impact on any project. I think it's safe to make it a `fix`, but a `break` would not hurt imo.

Reply:
yolo it is